packages/react/src/components/Accordion/AccordionItem.tsx

```diff
@@ -186,8 +161,10 @@ function AccordionItem({
           {title}
         </Text>
       </Toggle>
-      <div id={id} className={`${prefix}--accordion__content`}>
-        {children}
+      <div className={`${prefix}--accordion__wrapper`}>
+        <div id={id} className={`${prefix}--accordion__content`}>
+          {children}
+        </div>
```

Comment:
Is there a way to fix this without modifying the markup? This can break tests in downstream applications that rely on this component's markup to be stable between minor versions.

Reply:
Unfortunately, there isn't a way to fix this without modifying the markup. Let me explain why. The accordion content currently has padding, and if we animate only the content, the accordion won't completely close smoothly. You can see this in the GIF 

![Recording 2023-10-11 at 21 28 52](https://github.com/carbon-design-system/carbon/assets/25573926/3fc6824e-7e06-4d50-a4f7-59d5bd1bc4ed)

You can appreciate that there is a small gap, and it is due to the content padding. While it might be tempting to remove the padding when closing the accordion, it results in a less smooth and seamless animation. Here's an example of that approach in action 

![Recording 2023-10-11 at 21 43 30](https://github.com/carbon-design-system/carbon/assets/25573926/e5e1059b-d85d-4a6a-a652-a0f967af747d)

As you can observe, the animation looks bad because the content shifts due to the padding appearing and disappearing, resulting in a somewhat hacky solution.